tests/test-utils.js

```diff
@@ -33,3 +34,24 @@ export function objectifyEncodedReqBody(reqBody = '') {
 
   return paramsObject;
 }
+
+export const compareInnerHTMLWithIntlTranslation = (
+  assert,
+  { element, selector, message, doIncludesCheck }
+) => {
+  const innerHTML =
+    element?.innerHTML || find(selector.trim()).innerHTML.trim();
+
+  if (doIncludesCheck) {
+    assert.true(
+      innerHTML.includes(message),
+      `Element HTML: "${innerHTML}" <=====> Compared HTML: "${message}"`
```

Comment:
assert message can be defined to variable and re-used

Reply:
Updated